app/assets/javascripts/modules/tabs.js.coffee

```diff
@@ -1 +1,25 @@
+define('tabs', [], ->
+  class Tabs
+    constructor: (@$container) ->
+      @addEventListners()
 
+    addEventListners: ->
+      @$container.find('.js-tab-title').on('click', (e) =>
+        $clicked = e.target
+
+        @updateActiveTabTitle($clicked);
+        @updateActiveTabContent($clicked);
+      )
+
+    updateActiveTabTitle: (tabTitle) ->
+      @$container.find($('.js-tab-title')).removeClass('tab__title--active')
+      @$container.find(tabTitle).addClass('tab__title--active')
+
+    updateActiveTabContent: (id) ->
+      tabContentSelector = '[data-id="' + $(id).data('id') + '-content"]'
```

Comment:
with coffeescript you get string interpolation (all praise)! 

So this becomes `"[data-id='#{$(id).data('id')}-content']"`